include/rocksdb/utilities/stackable_db.h

```diff
@@ -92,6 +92,10 @@ class StackableDB : public DB {
                    const WideColumns& columns) override {
     return db_->PutEntity(options, column_family, key, columns);
   }
+  Status PutEntity(const WriteOptions& options, const Slice& key,
+                   const AttributeGroups& attribute_groups) override {
+    return db_->PutEntity(options, key, attribute_groups);
+  }
```

Comment:
One more thing we could consider is to add overrides to `DBImplReadOnly` / `DBImplSecondary` so they bail out early when the new API is called (seems to me it would happen anyways but only after some preliminary work has been done).